classes/abstracts/ActionScheduler_Abstract_ListTable.php

```diff
@@ -680,21 +680,27 @@ protected function display_filter_by_status() {
 			$this->status_counts = array( 'all' => $all_count ) + $this->status_counts;
 		}
 
-		foreach ( $this->status_counts as $status_name => $count ) {
+		// Translated status labels.
+		$status_labels             = ActionScheduler_Store::instance()->get_status_labels();
+		$status_labels['all']      = _x( 'All', 'status labels', 'action-scheduler' );
+		$status_labels['past-due'] = _x( 'Past-due', 'status labels', 'action-scheduler' );
```

Comment:
🤔 I get why the "All" label would be added here, but why wouldn't "Past-due" get added to the `get_status_labels()` method?

Reply:
I don't think we've defined a vocabulary for this, but both could possibly be said to be "meta-statuses" in the sense that neither can be found in the actions table (past-due actions are just a subset of those with pending status).

That's not necessarily a reason to exclude it from the `get_status_labels()` method, I was just slightly hesitant because the other statuses that method returns are 'concrete' statuses, each with a matching `ActionScheduler_Store::STATUS_*` constant. 

So ... I can't think of problems adding it would cause, but I wanted to err on the side of caution. Open to making a change, though.

Reply:
Ah ok, I didn't realize Past-due was also a "meta" status. I guess it probably makes sense to leave it out then, in case there's some other usage of `get_status_labels` that assumes they're all things you would save to the database or something.